x/payment/types/payformessage_test.go

```diff
@@ -257,105 +256,6 @@ func TestSignMalleatedTxs(t *testing.T) {
 	}
 }
 
-func TestWirePayForMessage_ValidateBasic(t *testing.T) {
```

Comment:
Moved to x/payment/types/wirepayformessage_test.go

Test there also:
 - is slightly simpler
 - not string based
 - ensures sdk errors / abci codes